lib/superform/rails/field.rb

```diff
@@ -0,0 +1,35 @@
+module Superform
+  module Rails
+    # The Field class is designed to be extended to create custom forms.
+    # Override the methods in the subclass to use your custom component classes.
+    class Field < Superform::Field
```

Comment:
Seeing as a Field node is a first class citizen to the Superform core, it feels out of place as inner class of the Form class that is not used in the Superform basic classes. Also causing issues in #50 , this feels like the right naming for Superform::Rails::Factory to create Superform::Rails::Field nodes.